### PR TITLE
HRIS-42 [Markup] Create Drawer which displays the list of downloaded files and remarks of the time entry

### PR DESCRIPTION
**Pull request by JohnRoyCabezas**
## Issue Link
[Backlog Link](https://framgiaph.backlog.com/view/HRIS-42)

## Definition of Done

- [x] Users can view the drawer which displays the list of downloaded files and remarks of the time entry

## Notes
- [x] Drawer is currently handled by shallow links. (In another PR)
- [x] Developers can only test it by manually changing the boolean code. True/False
- [x] Already indicated where devs can insert the data from the backend when integration comes.
## Pre-condition

cd client
npm run dev

## Expected Output

- [x] Users can view the drawer which displays the list of downloaded files and remarks of the time entry

## Screenshots/Recordings

image

### CODE REVIEW

File: client/src/middleware.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { NextResponse, NextRequest } from 'next/server'
 import { getToken } from 'next-auth/jwt'
 
 export default async function middleware(req: NextRequest): Promise<NextResponse | undefined> {
-  const session = await getToken({ req, secret: process.env.NEXT_PUBLIC_JWT_SECRET })
+  const session = await getToken({ req, secret: process.env.JWT_SECRET })
```
Comment (jsvelte) [inline client/src/middleware.ts:5]: We are experiencing infinite loop here due to your recent modification
image

Please change it to  NEXT_PUBLIC_JWT_SECRET

Comment (jsvelte) [reply client/src/middleware.ts:5]: in your `.env.local`, `.env.api` and in your middlware.ts

File: client/src/components/organisms/ViewDetailsDrawer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                  'resize-none text-sm font-normal text-slate-700 transition'
+                )}
+              >
+                <button
+                  type="button"
+                  className="flex w-full justify-between rounded px-2 py-2 text-xs font-medium text-slate-700 transition duration-150 ease-in-out focus:outline-none focus:ring-0 hover:bg-slate-700 hover:bg-opacity-5"
+                >
+                  <div className="flex">
+                    <div className="mr-1">
+                      <img src="https://files.slack.com/files-pri/T028JVBUY4F-F04J407U1H7/image.png"></img>
```
Comment (jsvelte) [inline client/src/components/organisms/ViewDetailsDrawer/index.tsx:140]: This image is broken please put the absolute url into the next.config.js

image

**Review by rogeliojohnoliverio — APPROVED** [review]
Approved 🙇

**Review by jsvelte — APPROVED** [review]
# Approved 💯

**Pull request merged**